test/blocks/table/table.test.js

```diff
@@ -9,7 +9,7 @@ describe('table and tablemetadata', () => {
   beforeEach(() => {
     const tables = document.querySelectorAll('.table');
     tables.forEach((t) => init(t));
-    window.dispatchEvent(new Event('milo:icons:loaded'));
+    window.dispatchEvent(new Event('milo:LCP:loaded'));
   });
```

Comment:
should you add any tests for some of the additional functionality added? re: table.js 311-319?